core/src/main/scala/org/apache/spark/ui/exec/ExecutorsTab.scala

```diff
@@ -170,6 +170,17 @@ class ExecutorsListener(storageStatusListener: StorageStatusListener, conf: Spar
     execTaskSummary.isBlacklisted = isBlacklisted
   }
 
+  def getExecutorHost(eid: String): String = {
```

Comment:
As the list active StorageStatusList stores the info of all the active executors at any point, I thought it could be used in that manner. I tested it and it seems to work fine. If you think there could be a potential issue with this approach, do let me know and we can discuss further.